### DISCUSSION TITLE
Customization of Azure Container Apps on Npm Apps

### DISCUSSION THREAD

**Discussion by DanielToft** [microsoft/aspire#6400] [General]
Hallo

In Version 9-rc we got a new feature where we can customize the container apps on projects with PublishAsAzureContainerApp. This feature is very cool, but is it supported in any way on NPM apps/executable resources?

**Comment by davidfowl** [comment]
You're right, this is only support for projects and containers. We're in the process of rethinking how we approach node/python/executables in general.

See https://github.com/dotnet/aspire/issues/4067#issuecomment-2416160621 for an approach that should work.

**Reply by DanielToft** [reply]
Thanks @davidfowl, this works great!
